apps/emqx_conf/src/emqx_conf_cli.erl

```diff
@@ -855,7 +857,7 @@ print_inconsistent_conf(New, Old, Options) ->
         target := {Target, TargetTnxId},
         node := {Node, NodeTnxId}
     } = Options,
-    emqx_ctl:print("~ts(tnx_id=~w)'s ~s is diff from ~ts(tnx_id=~w).~n", [
+    emqx_ctl:print("~ts(tnx_id=~w)'s ~s is different from ~ts(tnx_id=~w).~n", [
```

Comment:
Nit / Q: Why is it _tnx_ and not _txn_, which is a common abbreviation for _transaction_?

Reply:
Good question.  I can only guess.  Maybe who wrote this initially is not familiar with that convention, or it's a typo that got carried by copy-and-pasting.  :thinking: